build_historical_fallback.py

```
@@ -338,7 +395,11 @@ def build_parquet_from_wayback(
 
     rows: dict[pd.Timestamp, list[str]] = {}
     for date_str, tickers in snapshots:
-        ts = pd.Timestamp(date_str)
+        try:
+            ts = pd.Timestamp(date_str)  # FIX-14
+        except Exception as exc:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)